stable/aws-node-termination-handler/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 name: aws-node-termination-handler
 description: A Helm chart for the AWS Node Termination Handler
 version: 0.6.0
-appVersion: 1.2.0
```

Comment:
this version should not be updated, just the regular `version` field above it for helm.  Also, we have a pending PR for v1.3.0 release of node termination handler so this change should go in after that w/ version `0.7.1`

Reply:
fixed

Reply:
nice, should just need to rebase now.

Reply:
rebased

Reply:
looks like the Chart.yaml version got wiped away on rebase. If you update version to `0.7.1`, we should be good to go :)

Reply:
done

Reply:
awesome! thanks!